server/routes/products.js

```diff
@@ -0,0 +1,30 @@
+const Router = require("express");
+const Product = require("../models/Product");
+const router = new Router()
+
+const getProducts = async (req, res) => {
+    try {
+        const products = await Product.find()
+        
+        res.status(200).json(products)
+    } catch (error) {
+        res.status(404).json({ message: error.message })
+    } 
+}
+
+const sendProducts = async (req, res) => {
+    const product = req.body;
+
+    const newProduct = new Product(product)
+    try {
+    await newProduct.save()

```

Comment:
@AStashevskaya It calling `.save` is required, since we passed `product` to `constructor`?